### PR TITLE
About section

### PR DESCRIPTION
**Pull request by ArtemG22**
http://restaurant150223.surge.sh/

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -226,4 +226,71 @@ html {
...
+/******************************/
+/* ABOUT SECTION */
+/******************************/
+
+.about-section {
+  background: url('../gallery/banners/about-banner.png');
+  background-size: cover;
+  padding-top: 7.6rem;
+  padding-left: 29rem;
+  width: 160rem;
```
Comment (magicsd) [inline css/style.css]: Width is already specified at `.main-wrapper`. So this container should just take all the width available (100%)

File: css/style.css [hunk 2]
```diff
@@ -226,4 +226,71 @@ html {
...
+  height: 71rem;
```
Comment (magicsd) [inline css/style.css]: Height should not be specified, but determined automatically by content

File: css/style.css [hunk 3]
```diff
@@ -226,4 +226,71 @@ html {
...
+.our-mission {
+  position: relative;
+  width: 42rem;
+  padding-top: 2.2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  align-items: center;
+  text-align: center;
+  margin-right: 6.4rem;
```
Comment (magicsd) [inline css/style.css]: `.about-section`'s gap is expected to be used

File: css/style.css [hunk 4]
```diff
@@ -226,4 +226,71 @@ html {
...
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: linear-gradient(to top left, rgba(85, 85, 85, 0.2) -100%, transparent 70%);
+}
+
+.our-mission {
+  position: relative;
```
Comment (magicsd) [inline css/style.css]: What's the purpose?

File: css/style.css [hunk 5]
```diff
@@ -226,4 +226,71 @@ html {
...
+
+.about-section {
+  background: url('../gallery/banners/about-banner.png');
+  background-size: cover;
+  padding-top: 7.6rem;
+  padding-left: 29rem;
+  width: 160rem;
+  height: 71rem;
+  display: inline-flex;
+  justify-content: flex-start;
```
Comment (magicsd) [inline css/style.css]: I think it should be `center`

File: css/style.css [hunk 6]
```diff
@@ -226,4 +226,71 @@ html {
...
+/******************************/
+
+.about-section {
+  background: url('../gallery/banners/about-banner.png');
+  background-size: cover;
+  padding-top: 7.6rem;
+  padding-left: 29rem;
+  width: 160rem;
+  height: 71rem;
+  display: inline-flex;
```
Comment (magicsd) [inline css/style.css]: Why `inline-flex` but not `flex`?

File: css/style.css [hunk 7]
```diff
@@ -226,4 +226,71 @@ html {
...
+  font-family: "Lato", sans-serif;
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: #4c4c4c;
+  line-height: 2.4rem;
+  margin-bottom: 3.6rem;
+}
+
+.meal-img {
+  position: absolute;
```
Comment (magicsd) [inline css/style.css]: ???

File: css/style.css [hunk 8]
```diff
@@ -226,4 +226,71 @@ html {
...
+  height: 55.2rem;
```
Comment (magicsd) [inline css/style.css]: Setting the width is also important. BTW image size is `551px` x `551px`, not `552`

File: css/style.css [hunk 9]
```diff
@@ -226,4 +226,71 @@ html {
...
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: linear-gradient(to top left, rgba(85, 85, 85, 0.2) -100%, transparent 70%);
+}
+
+.our-mission {
+  position: relative;
+  width: 42rem;
+  padding-top: 2.2rem;
```
Comment (magicsd) [inline css/style.css:261]: I think if you position items in parent container to `center` you can drop this

File: css/style.css [hunk 10]
```diff
@@ -226,4 +226,71 @@ html {
...
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
```
Comment (magicsd) [inline css/style.css]: If you're using custom margins on container items, `gap` here is not expected to be used

File: index.html [hunk 11]
```diff
@@ -72,6 +72,22 @@ <h2 class="promo-slogan">The right ingredients<br>for the right food</h2>
         </div>
       </section>
     </main>
+
+    <section class="about-section">
+      <section class="our-mission">
+        <h3 class="slogan-our-mission">Just the right food</h3>
+        <img src="gallery/decoration/decoration-2.png" class="decoration-about" alt="decoration-about">
```
Comment (magicsd) [inline index.html]: Let's put valid `alt` text https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt

File: index.html [hunk 12]
```diff
@@ -72,6 +72,22 @@ <h2 class="promo-slogan">The right ingredients<br>for the right food</h2>
         </div>
       </section>
     </main>
+
+    <section class="about-section">
+      <section class="our-mission">
```
Comment (magicsd) [inline index.html]: Looks even like `figure` with `figcaption` and `blockquote`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#quotations

```html
<section class="about-section">
  <figure>
    <figcaption>
      <h3 />
      <img />
    </figcaption>
    <blockquote />
    <img />
  </figure>
  <figure />
</section>
```

File: index.html [hunk 13]
```diff
@@ -80,8 +80,8 @@ <h2 class="promo-slogan">The right ingredients<br>for the right food</h2>
       <figure class="our-mission">
         <figcaption>
           <h3 class="slogan-our-mission">Just the right food</h3>
-          <img src="gallery/decoration/decoration-2.png" class="decoration-about" alt="">
         </figcaption>
+        <img src="gallery/decoration/decoration-2.png" class="decoration-about" alt="">
```
Comment (magicsd) [inline index.html:84]: `alt` text is required for all images. If image not loaded user have to know what kind of image is this.

Comment (ArtemG22) [reply index.html:84]: ok

**Pull request merged**